heatmap_function.R

```diff
@@ -0,0 +1,90 @@
+library(shiny)
+library(shinydashboard)
+library(leaflet)
+library(leaflet.extras)
+library(sf)
+library(ggplot2)
+library(dplyr)
+library(lubridate)
+library(tidyverse)
+library(hms)
+library(RColorBrewer)
+library(classInt)
+
+
+##############Render Data##############
+
+render_heatmap_df <- function(data) {
+  dat2<-read_csv("data/Surrey_desc.csv")
```

Comment:
dsf already loaded on app startup, shouldn't need to be reloaded everytime the function is called